RFC/src/Glossary.md

```diff
@@ -96,6 +127,16 @@ MimbleWimble is a privacy-centric cryptocurrency protocol. It was
 [dropped](https://download.wpsoftware.net/bitcoin/wizardry/mimblewimble.txt) in the Bitcoin Developers chatroom by an
 anonymous author and has since been refined by several authors, including Andrew Poelstra.
 
+## RegistrationCollateral
+[RegistrationCollateral]: #registrationcollateral 'An amount of tari coin that is locked up on the base layer when a [Validator Node] is registered'
+
+In order to make Sybil attacks expensive and to provide an authorative base layer registry of [validator node]s they will need to lock up
+a amount of [Tari Coin] on the [Base Layer] using a registration transaction to begin acting as a VN on the DAN.
+
+## RegistrationPeriod
```

Comment:
The concept is great, but I wonder if `RegistrationTerm` better reflects what this concept refers to? 

There are some other assumptions here of course. `RegistrationTerm` assumed that the VN's registration expires and needs to be renewed (ala HTTPS certificates). 

That said, maybe we don't need to conflate the period that a node acts as a VN and the period that the collateral is locked up for. Maybe the best approach is to have two terms here: `CollateralPeriod` for the latter and `RegistrationTerm` for the former; these periods may or may not be equal.

Reply:
So I suppose that the registration will be valid for as long as the UTXO exists, so yes after the timelock has expired that doesn't mean the registration is over, it is only over when you spend the UTXO. I will try update the wording.

Reply:
Though on second reading I do define it that way, that this period is the MINIMUM period that a registration can last.